opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -831,7 +831,7 @@ namespace Opm
             const auto pmode_orig = ws.production_cmode;
             const auto imode_orig = ws.injection_cmode;
 
-            Scalar qtotal_orig = 0.0;
+            const Scalar qtotal_orig = 0.0;
```

Comment:
This will not work. I was referring to the above

```c++
const auto& ws = ...
```

Reply:
typo, now fixed